build/scripts/DevEnv/NewDevEnv.ps1

```diff
@@ -0,0 +1,189 @@
+using module .\AppProjectInfo.class.psm1
```

Comment:
Don't be shy with this script 😄 Should we put it in the root folder instead?

Reply:
That's because I'm still not sure if script is the better option here. _NewDevEnv_ is a script now because _localDevEnv_ is, but wouldn't a proper cmdlet be better? E.g. `New-BCDevEnv` 😎

Reply:
Yeah perhaps. But then we'd need some kind of init script in the root? To init the New-BCDevEnv command

Reply:
Keeping it as is right now. We can always restructure later.